src/components/RegionItem/RegionItem.tsx

```diff
@@ -13,21 +13,29 @@ import { Region, State } from 'common/regions';
 import { StyledRegionName } from 'components/SharedComponents';
 import { getSummaryFromFips } from 'common/location_summaries';
 import { EventAction, EventCategory, trackEvent } from 'components/Analytics';
-import { vaccineColorFromLocationSummary } from 'common/colors';
-import { summaryToStats } from 'components/NewLocationPage/SummaryStat/utils';
-import { Metric } from 'common/metricEnum';
-import { formatPercent } from 'common/utils';
+import { colorFromLocationSummary } from 'common/colors';
+import { Level } from 'common/level';
+
+function getLevelName(level: Level): string {
+  switch (level) {
+    case Level.LOW:
+      return 'Low';
+    case Level.MEDIUM:
+      return 'Medium';
+    case Level.HIGH:
+      return 'High';
+    default:
+      return 'Unknown';
+  }
```

Comment:
I think instead of duplicating this level => name mapping, we can use the `LOCATION_SUMMARY_LEVELS` const similar to how we do for above-the-fold location summary.
https://github.com/covid-projections/covid-projections/blob/develop/src/components/NewLocationPage/OverallRiskBlock/OverallLevelText/OverallLevelText.tsx#L31